src/Model/Verb.php

```diff
@@ -48,4 +48,24 @@ public static function getID($verb)
 
 		return DBA::lastInsertId();
 	}
+
+	/**
+	 * Return verb name for the given ID
+	 *
+	 * @param integer $id
+	 * @return string verb
+	 */
+	public static function getbyID(int $id)
```

Comment:
Small formatting change.
```suggestion
	public static function getByID(int $id)
```

Reply:
Ops, my shift key hadn't worked :-)